src/init.cpp

```diff
@@ -1554,8 +1562,6 @@ bool AppInit2(Config &config, thread_group &threadGroup)
     if (!strErrors.str().empty())
         return InitError(strErrors.str());
 
-    RandAddSeedPerfmon();
-
```

Comment:
maybe a dumb question but why take this out , don't we still need something here for a seed?

Reply:
This change has been ported from bitcoin/bitcoin/pull/7891.

The seed initialization has been moved closer to place where we actually use it, i.e. mainly private key generation.

To go in more detail  `RandAddSeedPerfmon()` ahs been included in `GetStrongRandBytes()` which is the function delegated to initialize the seed for the RNG. At the end of the port this initialization will use 3 sources: OpenSSL RNG, OS RNG and HW RNG (if available).